### PR TITLE
Swap cancellation collateral

### PR DESCRIPTION
**Pull request by zuzunker**

### CODE REVIEW

**Comment by zuzunker** [comment]
[YOEXT-1151: Add collateral handling to swap](https://emurgo.atlassian.net/browse/YOEXT-1151)

File: packages/yoroi-extension/app/api/ada/lib/storage/models/ConceptualWallet/index.js [hunk 1]
```diff
@@ -79,6 +79,13 @@ export class ConceptualWallet implements IConceptualWallet, IRename {
   getWalletType: void => WalletType = () => {
     return this.walletType;
   }
+
+  getWalletVariant: void => 'web' | 'ledger' | 'trezor' = () => {
```
Comment (yushih) [inline packages/yoroi-extension/app/api/ada/lib/storage/models/ConceptualWallet/index.js:83]: Only nitpicking: "mnemonic" seems more accurate than "web"

**Review by yushih — DISMISSED** [review]

File: packages/yoroi-extension/app/api/ada/lib/storage/models/ConceptualWallet/index.js (hunk 1 above)
Comment (zuzunker) [reply packages/yoroi-extension/app/api/ada/lib/storage/models/ConceptualWallet/index.js:83]: agreed, but that's just a refactoring of previously existing code so didn't want to break anything. Later can change. Especially after the merge of the DB PR

**Review by zuzunker — COMMENTED** [review]
/check

**Review by zuzunker — COMMENTED** [review]
/check

**Review by zuzunker — COMMENTED** [review]
/check

**Pull request merged**